etc/notes/CHANGES_5.0.0.md

```diff
@@ -18,6 +18,79 @@ The following is a detailed collection of the changes in the major v5 release of
 
 ### Optional callback support migrated to `mongodb-legacy`
 
+Node v5 drops support for callbacks in favor of a pure async-await API.  Below are some strategies for
+callback users to adopt driver v5 in recommendation order.
+
+### Migrate to promise based api (recommended!)
```

Comment:
Should this and the other subheadings in this section use the `####` instead of `###`?